contracts/OBSSStorage.sol

```diff
@@ -81,7 +85,25 @@ contract OBSSStorage is Ownable, ERC2771Recipient, Versioned {
     uint256 reactionId
   );
 
-  constructor(address _forwarder, string memory _version) Versioned(_version) {
+  // Modifiers
+  modifier onlyAllowedAddress() {
+    if (
```

Comment:
should it be `!vcAllowMap.isAddressAllowed(_msgSender()) || !founderAllowMap.isAddressAllowed(_msgSender()`?